Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,628 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+ select Population,Name from city where Name='Rotterdam';
```

Comment:
A convention, though not a rule, is to put the SQL keywords in capital letters - like `SELECT Population, Name FROM city`. It makes it easier to identify which words are column or table names and which words are SQL keywords.

Reply:
Also, be careful with where you insert spaces. I would suggest `Population, Name` instead of `Population,Name` as you've currently written.